_posts/2019-10-16-monthly-dev-post.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Monthly Dev Post October 2019
+author: alberth
+---
+
+Welcome to our Dev Post for October 2019.
+Every month (ahem) one of these post will be created, to tell you about the latest developments in the world of OpenTTD.
+
+<!-- more -->
+
+## Development
+
+### 1.9.3 Released
+
+OpenTTD [1.9.3 update](https://www.openttd.org/news/2019/09/16/openttd-1-9-3.html) was released in September.
+
+This wrapped up a bunch of bug fixes and and performance improvements, thanks to all contributors.
+
+Meanwhile [OpenTTD 1.10.0 is being prepared](https://github.com/OpenTTD/OpenTTD/pull/7726).  No release date is known yet, but find it here when it's announced :)
+
+### Updated SDL driver to SDL2.
+
+[http://www.libsdl.org/](Simple DirectMedia Layer) is a cross-platform development library designed to provide low level access to audio, keyboard, mouse, joystick, and graphics hardware via OpenGL and Direct3D.
+
+[Pull request #7086](https://github.com/OpenTTD/OpenTTD/pull/7086) updates to SDL2, which has been a long-sought upgrade.  This aids future compatibility.  It may also improve performance on some systems, but results on that are inconclusive at the moment.
+
+### Housekeeping
+
+Maintenance commits and small fixes continue.  In September and October 2019 these include updates to the OpenTTD compilation system, fixes to the midi audio driver for some systems, performance fixes and macOS and Windows 7 compatibility updates.
+
+A [game-crashing bug](https://github.com/OpenTTD/OpenTTD/pull/7755) with relocated station signs was fixed, and a [crash when exiting the scenario editor](https://github.com/OpenTTD/OpenTTD/commit/1e5029563cb68e53e41299a5d92e317566d7ba66).
+
+And of course translations continue to be updated.
```

Comment:
While starting a sentence with a conjunction (And) is fine, if you are going for a more formal tone, it could be rephrased slightly.